lib/rubocop/cop/rails/render_inline.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop looks for inline rendering within controller actions.
+      #
+      # @example
+      #   # bad
+      #   class ProductsController < ApplicationController
+      #     def index
+      #       render inline: "<% products.each do |p| %><p><%= p.name %></p><% end %>", type: :erb
+      #     end
+      #   end
+      #
+      #   # good
+      #   # app/views/products/index.html.erb
+      #   # <% products.each do |p| %>
```

Comment:
Minor: I think the ERB lines shouldn't be double-commented, only single?

Reply:
Yes, but `rake documentation_syntax_check` task fails to parse if not commented. I haven't find a better way than to comment those lines.

Reply:
Ah, ok.